src/components/FAQSection/QuestionFAQ/QuestionFAQ.tsx

```diff
@@ -0,0 +1,24 @@
+import { IPublication } from "@/utils/interfaces";
+import {
+  QuestionContainer,
+  IconWrapper,
+  SubSectionFAQ,
+  Summary,
+  Title,
+} from "./QuestionFAQ.styles";
+
+const QuestionFAQ = ({ data }: { data: IPublication }) => {
+  const { title, type } = data;
+
+  return (
+    <QuestionContainer>
+      <Summary>
+        <Title>{type}</Title>
+        <IconWrapper id="close" size={16} stroke-width={2} />
+      </Summary>
+      <SubSectionFAQ>{title}</SubSectionFAQ>
```

Comment:
We need to change this to use the {type} to {title} on the title area and on the SubSectionFAQ I think it's better if we could use a rich text area on the answer part (the SubSectionFAQ)